rasa/nlu/training_data/formats/rasa_yaml.py

```diff
@@ -29,6 +47,8 @@
 
 MULTILINE_TRAINING_EXAMPLE_LEADING_SYMBOL = "-"
 
+NLU_SCHEMA_FILE = "nlu/schemas/nlu.yml"
```

Comment:
would it make sense in the future to have a "rasa" schema file combining core, nlu, responses and domain (given that you can combine their contents in files)?

Reply:
Yes, I'm also thinking about that.. Will definitely do in the future.